server/services/interventionsService.ts

```diff
@@ -103,6 +104,33 @@ export default class InterventionsService {
     return sanitisedError
   }
 
+  serializeDeliusServiceUser(deliusServiceUser: DeliusServiceUser | null): ServiceUser {
+    if (!deliusServiceUser) {
+      return {} as ServiceUser
+    }
+
+    const currentDisabilities = deliusServiceUser.disabilities
+      .filter(disability => {
+        const today = new Date().toString()
+        return disability.endDate === '' || Date.parse(disability.endDate) >= Date.parse(today)
+      })
+      .map(disability => disability.disabilityType.description)
+      .join(', ')
+
+    return {
+      crn: deliusServiceUser.otherIds.crn,
+      title: deliusServiceUser.title,
+      firstName: deliusServiceUser.firstName,
+      lastName: deliusServiceUser.surname,
+      dateOfBirth: deliusServiceUser.dateOfBirth,
```

Comment:
I imagine that the interventions service would want to enforce a format here - i.e. ISO date format like we use for e.g. completion deadline. So looks like some translation might be required from the Delius data.

Reply:
Good point, I'll convert to ISO here like completion deadline.

Reply:
Turns out we can actually use the `CalendarDay` functionality here as-is, as I got the format back from Delius wrong, it's actually e.g. `1980-01-01`